File: experiments/finance/gnucash/swiftui/Dropdown_item_2lines.swift
import SwiftUI

struct Dropdown_item_2lines: View {
  var body: some View {
    VStack(alignment: .leading, spacing: 0) {
      Text("Transaction description").font(.system(size: 20)).foregroundColor(Color.black).multilineTextAlignment(.center)
      Text("Amount and date")
    }
    .frame(maxWidth: .infinity)
    .padding(.top, 5.0)
  }
}

struct Dropdown_item_2lines_Previews: PreviewProvider {
  static var previews: some View {
    Dropdown_item_2lines()
  }
}

File: experiments/finance/gnucash/compose/dropdown_item_2lines.kt
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.paint
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp

@Composable
fun dropdown_item_2lines() {
  Column(modifier = Modifier.weight(1.0f).padding(top = 5.0.dp)) {
    Text(text = "Transaction description", modifier = Modifier.wrapContentWidth().wrapContentHeight(), color = Color.Black, fontSize = 20.sp, textAlign = TextAlign.Center)
    Text(text = "Amount and date", modifier = Modifier.wrapContentWidth().wrapContentHeight())
  }
}


@Preview(showBackground = true)
@Composable
fun dropdown_item_2lines_preview() {
    MyApplicationTheme {
        dropdown_item_2lines()
    }
}
